src/CrawlTestLvl/private/world.cpp

```diff
@@ -36,7 +37,13 @@ CRAWL::WorldRule *World::initWorldRules() {
              {registerObject<CRAWL::DynamicWint>(), 1},
 
              {registerObject<Background>(), 1},
-             {registerObject<Day>(), 1}}}
+             {registerObject<Day>(), 1}}},
+        {2,
```

Comment:
```suggestion
        {500,
```
you try spawn second items part too near wit first spawn point. so this items cannot spawn on your world.